packages/nx-plugin/src/generators/component-playwright-spec/files/src/components/__componentName__/__componentName__.spec.tsx.template

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+import { test, expect } from '@playwright/experimental-ct-react';
+
+test.use({ viewport: { width: 500, height: 500 } });
+
+test('should work', async ({ mount }) => {
+  const component = await mount(<div>Learn React</div>);
```

Comment:
not sure if this is the best default ? maybe rendering the actual control would make more sense with the test failing. I'd rather avoid situations that folks generate tests that have no value but are passing.